src/components/overlays/pause.js

```diff
@@ -18,9 +19,9 @@ export class Pause extends Screen {
         this.addBackgroundItems();
         const parentKey = this.context.parentScreens.slice(-1)[0].scene.key;
         const isAboveSelectScreen = parentKey.includes("select");
-        const achievements = this.context.config.theme.game.achievements ? ["achievements"] : [];
+        const achievements = gmi.achievements.get().length ? ["achievements"] : [];
```

Comment:
I like this - so to disable achievements do you just supply an empty achievements config?

Reply:
yup